tests/unit/switch_utils.c

```diff
@@ -84,6 +84,10 @@ FST_TEST_BEGIN(is_file_path)
 {
 	switch_bool_t b = switch_is_file_path("{av_record_audio_only=true");
 	fst_requires(b == SWITCH_FALSE);
+	b = switch_is_file_path("{av_record_audio_only=true,future_json_var='{key=value}'");
+	fst_requires(b == SWITCH_FALSE);
+	b = switch_is_file_path("{av_record_audio_only=true,future_json_var='{key=value}'}");
+	fst_requires(b == SWITCH_FALSE);
```

Comment:
Should this be true as all bracket is enclosed.

Reply:
that made me wondering too.. but then i realized that `switch_is_file_path` checks whether the file begins with `/` or it contains `://` (after the content of {} is removed). so this is not a valid filename